bin/cron/create.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
+
+const Cli = require('structured-cli');
+const Cron = require('../../lib/cron');
+const Logs = require('../../lib/logs');
+const PrintCronJob = require('../../lib/printCronJob');
+const ValidateCreateArgs = require('../../lib/validateCreateArgs');
+const WebtaskCreator = require('../../lib/webtaskCreator');
+const _ = require('lodash');
+
+const CRON_AUTH_MIDDLEWARE = '@webtask/cron-auth-middleware';
+const CRON_AUTH_MIDDLEWARE_VERSION = '^1.2.1';
+
+const createCommand = require('../create');
+
+module.exports = Cli.createCommand('create', {
+    description: 'Create a cron webtask',
+    plugins: [require('../_plugins/profile')],
+    optionGroups: _.extend({}, createCommand.optionGroups, {
+        'Cron options': {
+            'no-auth': {
+                description: 'Disable cron webtask authentication',
+                dest: 'noAuth',
+                type: 'boolean',
+            },
+            schedule: {
+                description:
+                    'Either a cron-formatted schedule (see: http://crontab.guru/) or an interval of hours ("h") or minutes ("m"). Note that not all intervals are possible.',
+                type: 'string',
+                required: true,
+            },
+            state: {
+                description: "Set the cron job's state",
+                choices: ['active', 'inactive'],
+                defaultValue: 'active',
+                type: 'string',
+            },
+            tz: {
+                description: `An IANA timezone name (see: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), relative to which the cron schedule will be calculated. If not specified, the webtask cluster's default timezone is used.`,
+                metavar: 'TIMEZONE',
+                type: 'string',
+            },
+        },
+    }),
+    options: _.extend({}, createCommand.options, {}),
+    params: createCommand.params,
+    epilog: [
+        'Examples:',
+        '1. Create a webtask that runs every five minutes:',
+        '  $ wt cron schedule --schedule 5m ./sample-webtasks/hello-world.js',
```

Comment:
Is this help text correct?

Reply:
It most certainly is _not_. Great catch!

Reply:
Fixed in 6a3a0ac.